ch08.adoc

```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
-  int m_scan_indices(tp_scan) ;     
-  int m_time_scan_indices(tp_time_scan) 
-  int i_track_indices(tp_track) ;   // shared by tp_interpolation and time_interpolation 
-  int i_scan_indices(tp_scan) ;     
-  int i_time_scan_indices(tp_time_scan) 
-
-  // Tie points
-  float m_lat(tp_track, tp_scan) ;
-    m_lat : standard_name = "latitude" ;
-    m_lat : units = "degrees_north" ;
-  float m_lon(tp_track, tp_scan) ;
-    m_lon : standard_name = "longitude" ;
-    m_lon : units = "degrees_east" ;
-  float m_sen_azi_ang(tp_track, tp_scan) ;
-    m_sen_azi_ang : standard_name = "sensor_azimuth_angle" ;
-    m_sen_azi_ang : units = "degrees" ;
-  float m_sen_zen_ang(tp_track, tp_scan) ;
-    m_sen_zen_ang : standard_name = "sensor_zenith_angle" ;
-    m_sen_zen_ang : units = "degrees" ;
-  float m_sol_azi_ang(tp_track, tp_scan) ;
-    m_sol_azi_ang : standard_name = "solar_azimuth_angle" ;
-    m_sol_azi_ang : units = "degrees" ;
-  float m_sol_zen_ang(tp_track, tp_scan) ;
-    m_sol_zen_ang : standard_name = "solar_zenith_angle" ;
-    m_sol_zen_ang : units = "degrees" ;
-
-  float i_lat(tp_track, tp_scan) ;
-    i_lat : standard_name = "latitude" ;
-    i_lat : units = "degrees_north" ;
-  float i_lon(tp_track, tp_scan) ;
-    i_lon : standard_name = "longitude" ;
-    i_lon : units = "degrees_east" ;
-  float i_sen_azi_ang(tp_track, tp_scan) ;
-    i_sen_azi_ang : standard_name = "sensor_azimuth_angle" ;
-    i_sen_azi_ang : units = "degrees" ;
-  float i_sen_zen_ang(tp_track, tp_scan) ;
-    i_sen_zen_ang : standard_name = "sensor_zenith_angle" ;
-    i_sen_zen_ang : units = "degrees" ;
-  float i_sol_azi_ang(tp_track, tp_scan) ;
-    i_sol_azi_ang : standard_name = "solar_azimuth_angle" ;
-    i_sol_azi_ang : units = "degrees" ;
-  float i_sol_zen_ang(tp_track, tp_scan) ;
-    i_sol_zen_ang : standard_name = "solar_zenith_angle" ;
-    i_sol_zen_ang : units = "degrees" ;
+  // VIIRS I-Band Channel 01 and 04
+  float I01_radiance(track, scan) ;
+    I01_radiance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I01_radiance:standard_name = "toa_outgoing_radiance_per_unit_wavelength" ;
+    I01_radiance:units = "W m-2 sr-1 m-1" ;
+  float I01_reflectance(track, scan) ;
+    I01_reflectance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I01_reflectance:long_name = "reflectance" ;
+    I01_reflectance:units = "1" ;
+
+  float I04_radiance(track, scan) ;
+    I04_radiance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I04_radiance:standard_name = "toa_outgoing_radiance_per_unit_wavelength" ;
+    I04_radiance:units = "W m-2 sr-1 m-1" ;
+  float I04_brightness_temperature(track, scan) ;
+    I04_brightness_temperature:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I04_brightness_temperature:standard_name = "brightness_temperature" ;
+    I04_brightness_temperature:units = "K" ;
```

Comment:
Do we need four data variables in this example? They all have the span the same dimensions and have the same `tie_points` attribute value ....